docs/en/observability/synthetics-role-write.asciidoc

```diff
@@ -94,10 +95,6 @@ If the user should be able to create and update monitors hosted on
 | `Fleet`: `All`
 | Access to Fleet in {kib}.
 
-| {kibana-ref}/kibana-privileges.html[Kibana]
-| `Integrations`: `All`
-| Access to Integrations in {kib}.
-
```

Comment:
we will need to revert this part. i just test on cloud for POST FF and user needs at least read permission as well to access fleet UI to create an agent policy.

Reply:
https://github.com/elastic/observability-docs/pull/3044 raised